applications/GeoMechanicsApplication/custom_processes/apply_write_result_scalar_process.hpp

```diff
@@ -106,14 +106,14 @@ class ApplyWriteScalarProcess : public Process
                 } else {
                     // open a new file and overwrite
                     mOutFile[i].open(fileName, std::ios::trunc); // overwrite
-                    mOutFile[i] << "Time" << "   " << mVariableName << std::endl;
-                    double value = it->FastGetSolutionStepValue(var);
-                    mOutFile[i] << Time << "   " << value << std::endl;
+                    mOutFile[i] << "Time" << "   " << mVariableName << "\n";
+                    const double &value = it->FastGetSolutionStepValue(var);
```

Comment:
```suggestion
                    const double value = it->FastGetSolutionStepValue(var);
```

simple types should be used without reference if unmutable